docs/contributing.rst

```diff
@@ -67,6 +67,8 @@ The following tools are there to help you:
 
 Integration
 ~~~~~~~~~~~
+Mac OS X users: Run `./tests/mac-integration.sh` to configure the
```

Comment:
@kuba. I put the sudo statement inside the script because it is only needed for modifying /etc/hosts which only needs to be done on the first run.

@davidxia. Good catch!